cip-0020.md

```diff
@@ -62,6 +61,10 @@ Unlike other blockchains, BURST does not have its own colored coin for running A
 
 The change will require a hard-fork. Any ATs created before the fork will be subject to the previous set of fees and will operate without any change. Only ATs created after the fork will run under the new fees.
 
+## Additional notes
+
+Upon activation, this CIP will be implemented in BRS and the changes to fee structure will take effect during the next hard fork.
+
```

Comment:
Can we add a block high from here for in 2 months or so ?

Reply:
Block height for next fork will be determined once we have laid out the full extent of our plans for that fork and, more importantly, have network consensus.

Reply:
oaky